tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -191,17 +191,17 @@ static PyObject * THPVariable_contiguous(PyObject* self, PyObject* args)
   END_HANDLE_TH_ERRORS
 }
 
-static Tensor dispatch_copy_(Tensor & self, const Tensor & other, bool async) {
+static Tensor dispatch_copy_(Tensor & self, const Tensor & other, bool non_blocking) {
   AutoNoGIL no_gil;
   AutoGPU auto_gpu(self);
-  return self.copy_(other, async);
+  return self.copy_(other, non_blocking);
 }
 
 static PyObject * THPVariable_copy_(PyObject* self, PyObject* args, PyObject* kwargs)
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "copy_(Tensor other, bool async=False)"
+    "copy_(Tensor other, bool non_blocking=False)"
```

Comment:
When we merge Variable and Tensor,  I think this will be the only entry point for `copy_`. I think you can just add another entry for async. i.e.:

```c++
  static PythonArgParser parser({
    "copy_(Tensor other, bool non_blocking=False)",
    "copy_(Tensor other, bool async=False)|deprecated"
  });

Reply:
Alrighty, thanks. I also added such a line for `type_`.